src/ring_theory/polynomial/hermite/basic.lean

```diff
@@ -127,4 +132,69 @@ end
 
 end coeff
 
+section coeff_explicit
+
+open_locale nat
+
+/-- Because of `coeff_hermite_of_odd_add`, every nonzero coefficient is described as follows. -/
+lemma coeff_hermite_explicit :
```

Comment:
Could we just call this `coeff_hermite`, given that doesn't seem to otherwise exist? Not particularly concerned, just the `_explicit` isn't saying much.

Reply:
Another alternative would be to add a globally defined `coeff_hermite` that has an `if` statement.

Reply:
Sure - I'll do the latter. I'm not sure what's going to be more useful in practice, but I do know it was a bit annoying going between these definitions, so hopefully this saves someone some work down the road.

Reply:
I think `coeff_hermite` is better.